### PR TITLE
[16.0][MIG] project_task_pull_request: Migration to 16.0

### PR DESCRIPTION
**Pull request by valjaro**

### CODE REVIEW

**Comment by JSebastian7913** [comment]
Functional validation done.

**Review by NICO-SOLUTIONS — COMMENTED** [review]
Functional review 
looks good

**Comment by ivs-cetmix** [comment]
Hey @OCA/project-service-maintainers any chances to have it merged?

File: project_task_pull_request/__manifest__.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 {
     "name": "Project Task Pull Request",
     "summary": "Adds a field for a PR URI to project tasks",
-    "version": "15.0.1.0.1",
+    "version": "16.0.1.0.1",
```
Comment (pedrobaeza) [inline project_task_pull_request/__manifest__.py:7]: 16.0.1.0.0

**Comment by ivs-cetmix** [comment]
Hi @valjaro thank you for your contribution! Could you please fix the mentioned issues do we could have this PR merged?

File: project_task_pull_request/__manifest__.py (hunk 1 above)
Comment (ivs-cetmix) [reply project_task_pull_request/__manifest__.py:7]: hi @pedrobaeza is this the only one issue here? If yes and @valjaro is not available any longer would't you mind us reopening this PR from our own branch so we could have it merged?

Comment (pedrobaeza) [reply project_task_pull_request/__manifest__.py:7]: Yeah, it seems so.

**Comment by pedrobaeza** [comment]
Superseded by #1208

**Pull request closed without merging**